week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -0,0 +1,84 @@
+console.log("test");
+
+var myBlueBtn = document.getElementById("blueBtn");
+myBlueBtn.addEventListener("click", changeBlue, true);
+
+function changeBlue() {
+  //   alert("Blue Button Clicked");
+  var myDiv = document.querySelector(".Jumbotron");
+  myDiv.style.backgroundColor = "#588fbd";
+
+  var myBtn1 = document.querySelector(".btn.btn-primary.btn-lrg");
+  myBtn1.style.backgroundColor = "#ffa500";
+  //
+  var myBtn2 = document.querySelector(".btn.btn-secondary.btn-lrg");
+  myBtn2.style.backgroundColor = "black";
+  myBtn2.style.color = "white";
+  //   myBtn1.TEXT_NODE.style.backgroundColor = "white";
+}
+var myOrangeBtn = document.getElementById("orangeBtn");
+myOrangeBtn.addEventListener("click", changeOrange, true);
+
+function changeOrange() {
+  //   alert("Blue Button Clicked");
+  var myDiv = document.querySelector(".Jumbotron");
+  myDiv.style.backgroundColor = "#f0ad4e";
+
+  var myBtn1 = document.querySelector(".btn.btn-primary.btn-lrg");
+  myBtn1.style.backgroundColor = "#5751fd";
+  //
+
+  var myBtn2 = document.querySelector(".btn.btn-secondary.btn-lrg");
+  myBtn2.style.backgroundColor = "#31b0d5";
+  myBtn2.style.color = "white";
+}
+
+var myGreenBtn = document.getElementById("greenBtn");
+myGreenBtn.addEventListener("click", changeGreen, true);
+
+function changeGreen() {
+  //   alert("Blue Button Clicked");
+  var myDiv = document.querySelector(".Jumbotron");
+  myDiv.style.backgroundColor = "#87ca8a";
+
+  var myBtn1 = document.querySelector(".btn.btn-primary.btn-lrg");
+  myBtn1.style.backgroundColor = "black";
+  //
+
+  var myBtn2 = document.querySelector(".btn.btn-secondary.btn-lrg");
+  myBtn2.style.backgroundColor = "#8c9c08";
```

Comment:
Nice work, all buttons work correctly! However I can see some logic repetition here. You could select `myBtn1` and `myBtn2` just once outside your functions. Also all three functions make very similar changes. Can you think of the way how you could refactor the code so there is only one function responsible for making these changes?